### PR TITLE
Improve translatable strings containing "%s" to have a translator comment

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
When we use %s in a string and it is not totally obvious what %s means when reading the rest of the string, we should provide a translator comment.

Tries to address: https://github.com/WordPress/gutenberg/issues/5361

## How has this been tested?
Only comments were added to the translations was added in this changes, some smoke testing, and the automated tests should be enough to catch problems.

### CODE REVIEW

**Review by ocean90 — DISMISSED** [review]
The use of `_x()` is incorrect here. The context function is used to distinguish the same string used in different contexts. For additional info, like explaining placeholders, a translators comment should be added.

**Comment by jorgefilipecosta** [comment]
Thank you for explanation @ocean90, the code was updated to use translator comments.

File: components/server-side-render/index.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ export class ServerSideRender extends Component {
 				<Placeholder><Spinner /></Placeholder>
 			);
 		} else if ( response.error ) {
+			// translators: %s: error message describing the problem
 			return (
 				<Placeholder>{ sprintf( __( 'Error loading block: %s' ), response.errorMsg ) }</Placeholder>
```
Comment (afercia) [inline components/server-side-render/index.js]: Not sure if in JS it's the same, but ideally the translators comment should be immediately before the line where it is used e..g.:
```
<Placeholder>{ sprintf( 
    // translators: %s: error message describing the problem
    __( 'Error loading block: %s' )
```

@ocean90 ?

Comment (jorgefilipecosta) [reply components/server-side-render/index.js]: Hi @afercia thank you for referring this, the code was updated now translator comments are immediately before the line where they are used.

File: editor/utils/editor-media-upload/index.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default function editorMediaUpload( {
 	const errorHandler = ( { file, sizeAboveLimit, generalError } ) => {
 		let errorMsg;
 		if ( sizeAboveLimit ) {
+			// translators: %s: file name
```
Comment (gziolo) [inline editor/utils/editor-media-upload/index.js]: You probably need to move it inside `sprintf` just before the line with `__( ... )`.

**Review by gziolo — APPROVED** [review]
LGTM 👍 
There is one thing where we might need to move the comment closer to the translation function.

**Comment by jorgefilipecosta** [comment]
Thank you for the reviews!

**Pull request merged**